### ISSUE TITLE
Create tombstone pages for removed items

### ISSUE THREAD

**Issue by vantuyls** [osulp/Scholars-Archive#1801]
### Descriptive summary
As part of our promise of persistency for works in SA@OSU, we need to implement tombstone pages for works that have been removed from the repository. This should be a page that the URI for the item resolves to with text to indicate that the item in question has been removed or withdrawn from the system.

**Comment by vantuyls** [comment]
UNC has tombstoning here: https://github.com/UNC-Libraries/hy-c/pull/21

**Comment by straleyb** [comment]
@simholt @shieldsb @clarallebot What are the requirements for the application of this process. Do we want people to be able to tombstone their own works or do we want people to have to request to tombstone their works and then have an admin approve that request? 

This will effect our implementation of this feature greatly.

**Comment by clarallebot** [comment]
The second one, I think. The process to remove a work from ScholarsArchive@OSU is not straightforward (see https://wiki.library.oregonstate.edu/confluence/display/RP/ScholarsArchive@OSU+Policies#ScholarsArchive@OSUPolicies-Withdrawal,Replacement,andAccessRestrictionPolicy), and it must be approved by several people. I think that admins are the only ones that should be able to set up a tombstone.

**Comment by simholt** [comment]
I clicked the Tombstone button on a few items and each gave me the broken message. I had to go back into Works to locate them and add info to the text box.

So when you click Delete on an item, a box pops up asking if you really want to do that. I think that should happen with Tombstoning so we can prevent accidents.

<img width="318" alt="Screen Shot 2020-03-09 at 11 17 06 AM" src="https://user-images.githubusercontent.com/33705514/76244551-88e8dc80-61f7-11ea-8bff-e14bc825905b.png">
